feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -122,8 +122,8 @@ func (d *dutData) Configure(t *testing.T, dut *ondatra.DUTDevice) {
 		dut.Config().Interface(ocName).Replace(t, a.NewInterface(ocName))
 	}
 	dutBGP := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).
-		Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
-	dutBGP.Replace(t, d.bgpOC)
+		Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()	
+	dutBGP.Replace(t, d.bgpOC)	
```

Comment:
I don't see what's changed here besides the redundant whitespace?

Reply:
Fixed it. Thanks.